spec/liquid_tags/twitch_tag_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+require "nokogiri"
+
+RSpec.describe TwitchTag, type: :liquid_tag do
+  let(:slug) { "CuteSpicyNostrilDoritosChip" }
+
+  def assert_parses(slug, token)
+    liquid = Liquid::Template.parse("{% twitch #{token} %}")
+    html   = Nokogiri.parse(liquid.render).root
+    expect(html.name).to eq "iframe"
+    expect(html[:src]).to eq "https://clips.twitch.tv/embed?autoplay=false#{"&clip=" + slug + "&parent=localhost"}"
+    expect(html[:width]).to eq "710"
+    expect(html[:height]).to eq "399"
+  end
+
+  it "accepts twitch clip slug" do
+    assert_parses slug, slug
+  end
+
+  it "accepts twitch clip slug with wonky whitespace" do
+    assert_parses slug, " #{slug}  \t"
+  end
```

Comment:
```suggestion
  end

  it "forbids inserting autoplay option" do
    assert_parses slug, "#{slug}&autoplay=true"
  end

  it "allows inserting mute option" do
    assert_parses slug, "#{slug}&muted=true" # this needs fixing
  end
```

Reply:
The slug here would be the 'SpoopySlipperyGrasshopperPogChamp' the second param, token would be the user input.